streams.json

```diff
@@ -2314,44 +3062,78 @@
               "dlloyd@redhat.com"
           ],
           "repository_type": "GIT",
-          "repository_url": "https://github.com/wildfly/wildfly-naming-client",
+          "repository_url": "https://github.com/wildfly/wildfly-naming-client.git",
           "codebase": "master",
-          "tag": "1.0.7.Final",
+          "tag": "jboss-parent-19",
+          "group_id": "org.wildfly",
+          "artifact_id": "wildfly-naming-client",
           "version": "1.0.7.Final",
-          "gav": "org.wildfly",
+          "gav": "wildfly-naming-client:org.wildfly",
+          "comment": ""
+        },
+        {
+          "component_name": "wildfly-wildfly-openssl",
+          "contacts": [],
+          "repository_type": "GIT",
+          "repository_url": "https://github.com/wildfly/wildfly-openssl.git",
+          "codebase": "",
+          "tag": "unknown",
+          "group_id": "org.wildfly.openssl",
+          "artifact_id": "wildfly-openssl-parent",
+          "version": "1.0.2.Final",
+          "gav": "wildfly-openssl-parent:org.wildfly.openssl",
+          "comment": ""
+        },
+        {
+          "component_name": "wildfly-wildfly-transaction-client",
+          "contacts": [],
+          "repository_type": "GIT",
+          "repository_url": "https://github.com/wildfly/wildfly-transaction-client.git",
+          "codebase": "",
+          "tag": "jboss-parent-21",
+          "group_id": "org.wildfly.transaction",
+          "artifact_id": "wildfly-transaction-client",
+          "version": "1.0.2.Final",
+          "gav": "wildfly-transaction-client:org.wildfly.transaction",
           "comment": ""
         },
         {
           "component_name": "wsdl4j",
           "contacts": [],
           "repository_type": "CVS",
-          "repository_url": "cvs://pserver:anonymous:@wsdl4j.cvs.sourceforge.net:/cvsroot/wsdl4j",
+          "repository_url": ":pserver:anonymous:@wsdl4j.cvs.sourceforge.net:/cvsroot/wsdl4j wsdl4j",
```

Comment:
`:pserver:anonymous:@wsdl4j.cvs.sourceforge.net:/cvsroot/wsdl4j wsdl4j` is not correct.

`java.net.URISyntaxException: Expected scheme name at index 0:` for creating URI from it
`java.net.MalformedURLException: no protocol:` for creating URL from it.

Reply:
@soul2zimate Fixed